### PR TITLE
chore: format code with ReSharper after push

### PR DESCRIPTION
**Pull request by Epictek**
[[中文版模板 / Chinese template](https://github.com/ant-design-blazor/ant-design-blazor/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [x] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution

Github action to format  code after every push, example: https://github.com/Epictek/ant-design-blazor/commit/1d62190316cc2535c8a019ec4e86f5537dac0435

We might want to tweak the default settings before merging?

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: components/form/FormItem.razor [hunk 1]
```diff
@@ -6,8 +6,9 @@
     @if (!string.IsNullOrEmpty(Label))
     {
         <AntDesign.Col @attributes="GetLabelColAttributes()" class=@($"{_prefixCls}-label")>
-            <label class=@(Required ? $"{_prefixCls}-required" : _labelCls)> @(Label)</label>
-        </AntDesign.Col>
+        <label class=@(Required ? $"{_prefixCls}-required" : _labelCls)> @(Label)</label>
+        </AntDesign.
```
Comment (Epictek) [inline components/form/FormItem.razor:10]: Would seem like the formatter is breaking some of the Razor pages, will investigate this

Comment (diegofrata) [reply components/form/FormItem.razor:10]: ReSharper is absolute shit when it comes to formatting Razor files. Is there a way you can limit it to .cs files?

Comment (Epictek) [reply components/form/FormItem.razor:10]: That's a shame, I chose ReSharper for it's ability to format Razor files :( Guess I will investigate using dotnet format instead

**Comment by Epictek** [comment]
Will close this for now and replace it with a [dotnet format](https://github.com/dotnet/format) prehook that only runs on .cs files

**Pull request closed without merging**